addons/mail/controllers/discuss/channel.py

```diff
@@ -105,7 +105,7 @@ def discuss_channel_messages(self, channel_id, fetch_params=None):
             messages.set_message_done()
         return {
             **res,
-            "data": Store(messages).get_result(),
+            "data": Store(messages, messages._to_store_defaults(Store.Target())).get_result(),
```

Comment:
I don't think we need diff here anymore?
The correct target is automatically passed to `_to_store_defaults` internally.